Docs/source/usage/parameters.rst

```diff
@@ -1229,17 +1229,19 @@ Numerics and algorithms
     computational medium, respectively. The default values are the corresponding values
     in vacuum.
 
-* ``interpolation.nox``, ``interpolation.noy``, ``interpolation.noz`` (`1`, `2`, or `3` ; default: 1)
-    The order of the shape factors for the macroparticles, for the 3 dimensions of space.
-    Lower-order shape factors result in faster simulations, but more noisy results,
+* ``interpolation.shape_factors_order`` (`1`, `2`, or `3`)
+    The order of the shape factors for the macro-particles along all spatial directions.
+    Low-order shape factors result in faster simulations, but may lead to more noisy results.
+    High-order shape factors are computationally more expensive, but may increase the overall accuracy of the results. For production runs it is probably safer to use high-order shape factors, such as cubic order (that is, cubic splines).
```

Comment:
```suggestion
    High-order shape factors are computationally more expensive, but may increase the overall accuracy of the results. For production runs it is generally safer to use high-order shape factors, such as cubic order (that is, cubic splines).
```
I would suggest to use `generally` here, to emphasize that in almost all cases the user should choose `3` here!